homeassistant/components/wled/__init__.py

```diff
@@ -89,7 +91,8 @@
     cancel_timer()
 
     # Unload entities for this entry/device.
-    await hass.config_entries.async_forward_entry_unload(entry, LIGHT_DOMAIN)
+    for component in LIGHT_DOMAIN, SWITCH_DOMAIN:
+        await hass.config_entries.async_forward_entry_unload(entry, component)
```

Comment:
We can use `asyncio.gather` here to await the tasks concurrently.

Reply:
That is surely an improvement, which I think, we should make on many other places in our codebase.

Reply:
Done.